Gemfile

```diff
@@ -7,11 +7,3 @@ gem "jquery-rails"
 
 # Airbrake must be required before initialization because its middleware is added via an initializer
 gem 'airbrake'
-gem 'rollbar'
-
-group :development, :test do
-  gem 'pry'
-  gem 'pry-nav'
-  gem 'pry-rescue'
-  gem 'pry-stack_explorer'
```

Comment:
why did all the pry stuff get taken out? I feel like we have those in our other `:development, :test` groups